src/functions/views/provider.php

```diff
@@ -128,10 +128,11 @@ function tribe_events_widgets_v2_is_enabled() {
 	}
 
 
-	$env_var = getenv( 'TRIBE_EVENTS_WIDGETS_V2_ENABLED' );
+	$env_var = getenv( 'TRIBE_EVENTS_WIDGETS_V2_DISABLED' );
 	if ( false !== $env_var ) {
-		return (bool) $env_var;
+		return !! $env_var;
```

Comment:
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space
* :no_entry_sign: Expected 1 space before "!"; 0 found

<sub>hash: b2567e077435aa407806bc0e5ee2be72</sub>